contrib/inventory/puppetdb.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python2
+#
+# Ansible inventory script based on puppetdb.
+#
+# Author: Jan Collijs
+# Source: https://github.com/visibilityspots/ansible-puppet-inventory
+
+# If you want to use the api version 4 you have to install the pypuppetdb
+# library made by https://github.com/NeCTAR-RC/pypuppetdb from source untill
+# it got merged https://github.com/puppet-community/pypuppetdb/pull/34
+
+import sys
+import os
+import ConfigParser
+import argparse
+from pypuppetdb import connect
+try:
+        import json
+except:
+        import simplejson as json
+
+''' This class will generate a list of nodes based on puppetdb '''
+class PuppetDBInventory():
+    def __init__(self):
+        self.inventory = {}
+        self.read_settings()
+        self.parse_cli_args()
+        if self.args.list:
+            if self.puppetdb_api_version == '4':
+                data = self.get_host_list_based_on_environments()
+            else:
+                data = self.get_host_list()
+
+            print json.dumps(data)
```

Comment:
This line is responsible for the python 3.5 syntax error. Change it to the following to fix:

```python
print(json.dumps(data))
```